### DISCUSSION TITLE
Improve Build Cycle for capacitor App

### DISCUSSION THREAD

**Discussion by workstreamautomation** [ionic-team/capacitor#4732] [Ideas]
Hello Great people i just want to know is there any way to improve our build cycle for android and ios for our capacitor app as we are only changing javascript/typescript basically resources and not the native code so why we have to build every time we change something in the code.Is there any possibility to debug changes in real time in your physical device without building all sources.

**Comment by jcesarmobile** [comment]
You can use live reload feature

https://capacitorjs.com/docs/guides/live-reload
